lib/content-services/src/lib/dialogs/folder.dialog.spec.ts

```diff
@@ -302,4 +302,18 @@ describe('FolderDialogComponent', () => {
             });
         });
     });
+
+    it('should disable the submit button after submitting the form', () => {
```

Comment:
I think this test case should have an additional step to fill the name of the folder in and then perform submit, preferably by emulating the button click instead of calling the submit directly from the component. In this case we will be sure that button was enabled and then got disabled by submitting, right now the button is disabled by default when dialog is opened

Reply:
Thanks for catching!
I refactored the tests and now we have both tests: button click and component submit call. I also reorganized the tests a bit to reduce the amount of duplicate code, please take a look